math/.gitignore

```diff
@@ -1,24 +0,0 @@
-/target
-/lib
-/classes
-/checkouts
-pom.xml
-pom.xml.asc
-*.jar
-*.class
-.lein-deps-sum
-.lein-failures
-.lein-plugins
-.lein-repl-history
-*.swo
-db/
-.lein-env
-.nrepl-port
-scratch
-src/polismath/play.clj
-old
-.gorilla-port
-.lein-git-deps/hiphip
-wiki
-.lein-git-deps/debug-repl
-data
```

Comment:
A lot of this is old and can be removed, but I do think we want to keep these here.